components/navigation/navChild.js

```diff
@@ -48,20 +49,24 @@ const NavChild = ({ slug, page, color, className }) => {
 
   if (page.children?.length > 0 && visibleItems.length > 0) {
     accordion = (
-      <div className={styles.Accordion}>
+      <div className={`${styles.Accordion} -ml-[${55 + 24}px] pl-[${55 + 24}px]`}>
```

Comment:
why this weird calculations? `${55 + 24}px`

Reply:
made changes